src/web/praxisIsncsciCell/praxisIsncsciCell.ts

```diff
@@ -50,7 +50,7 @@ export class PraxisIsncsciCell extends HTMLElement {
       return;
     }
 
-    if (name === 'message') {
+    if (name === 'message' && this.messageElement) {
```

Comment:
Added this check as code failed after I set `strict = true` on the `tsconfig` file.